### PR TITLE
Create LIT7055Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I assumed, that this Qene is of Wāzema type, because it has five verses as [LIT7047Qene](https://github.com/BetaMasaheft/Works/pull/1165/files).

### CODE REVIEW

File: new/LIT7055Qene.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7055Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ ፋሲለደስ፡ መለኮትሰ፡ </title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7055Qene.xml:8]: Please drop መለኮትሰ፡, it belongs to the next phrase.

File: new/LIT7055Qene.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the Wāzemā-type. Type and author are not indicated in the manuscript.</p>
```
Comment (DenisNosnitsin1970) [inline new/LIT7055Qene.xml:37]: Yes, it seems to be wazema.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7055Qene.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7055Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ ፋሲለደስ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Lǝgʷāma wa-qǝnwāta ba-ʾamān tawakkǝfo Fāsiladas (Qǝne of wāzemā type)</title>
```
Comment (karljonaskarlsson) [inline new/LIT7055Qene.xml:9]: qǝnnǝwāta, tawakfa

File: new/LIT7055Qene.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            <change who="CH" when="2024-08-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rc. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ <persName ref="PRS4015Fasilada">ፋሲለደስ፡</persName> መለኮትሰ፡ ልጓመ፡ ዘይቤ፡ ውስተ፡ ገጽየ፡ ፈነወ፡ ኢዮብ፡ በእንተ፡ ደዊሁ፡ ኅዳጠ፡ ሐሰወ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7055Qene.xml:61]: ደዌየ

File: new/LIT7055Qene.xml [hunk 5]
```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7055Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ ፋሲለደስ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Lǝgʷāma wa-qǝnwāta ba-ʾamān tawakkǝfo Fāsiladas (Qǝne of wāzemā type)</title>
+                <title xml:lang="en" corresp="#t1">Fāsiladas in true faith receiving the halter and the spurs ...</title>
```
Comment (karljonaskarlsson) [inline new/LIT7055Qene.xml:10]: received

**Review by karljonaskarlsson — COMMENTED** [review]
Sorry for commenting after the merging

**Comment by CarstenHoffmannMarburg** [comment]
> Sorry for commenting after the merging

Thank you for checking the record and the text. I will make the updates in a new branch. This is absolutely no problem!

File: new/LIT7055Qene.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7055Qene.xml:61]: I see ደዊሁ፡/ደዌሁ፡ in the first verse. You mean, I should change to ደዌሁ፡ ?

Screenshot 2024_Qene7055 (II)

Comment (karljonaskarlsson) [reply new/LIT7055Qene.xml:61]: Yes, sorry! If you compare it to the "wi" two lines down, I think it's clear that it is "we" (which is also what we would expect, I suppose)

**Comment by karljonaskarlsson** [comment]
> > Sorry for commenting after the merging
> 
> Thank you for checking the record and the text. I will make the updates in a new branch. This is absolutely no problem!

Thank you :)

**Pull request merged**